openedx/features/enterprise_support/tests/test_api.py

```diff
@@ -245,6 +338,76 @@ def test_get_consent_required_courses(self, mock_catalog_contains_course):
         consent_required = get_consent_required_courses(user, [course_id])
         self.assertNotIn(course_id, consent_required)
 
+    def test_get_consent_required_courses_no_enterprise_user(self):
```

Comment:
nit: Probably not needed, but might be helpful to add a comment along the lines of "Verifies that consent is not required for a non-enterprise user"